src/library_manager/lib_manager.c

```diff
@@ -963,13 +963,15 @@ static int lib_manager_setup(uint32_t dma_id)
 
 	dma_block_cfg.dest_address = dma_ext->dma_addr;
 
+#if CONFIG_KCPS_DYNAMIC_CLOCK_CONTROL
 	/*
 	 * make sure that the DSP is running full speed for the duration of
 	 * library loading
 	 */
 	ret = core_kcps_adjust(cpu_get_id(), CLK_MAX_CPU_HZ / 1000);
 	if (ret < 0)
 		goto err_dma_buffer;
+#endif
```

Comment:
maybe a dummy inline `core_kcps_adjust()` implementation instead?

Reply:
@lyakh Tried this approach, but it gets a bit ugly as CLK_MAX_CPU_HZ is not defined either. Or we want to drop the requirement to define such values on SOF side.